### PR TITLE
Update recently changed python code to use python 2.6/2.7 constructs.

### PR DESCRIPTION
**Pull request by ericvaandering**
Done with 

`futurize -1 -nw -x print_with_import .`

### CODE REVIEW

File: Alignment/MillePedeAlignmentAlgorithm/scripts/mps_alisetup.py [hunk 1]
```diff
@@ -19,6 +19,7 @@
 import collections
 import Alignment.MillePedeAlignmentAlgorithm.mpslib.Mpslibclass as mpslib
 from Alignment.MillePedeAlignmentAlgorithm.alignmentsetup.helper import checked_out_MPS
+from functools import reduce
```
Comment (ghellwig) [inline Alignment/MillePedeAlignmentAlgorithm/scripts/mps_alisetup.py]: @ericvaandering does this imply a functional change of the `reduce` function?

**Comment by ghellwig** [comment]
please test

File: Alignment/MillePedeAlignmentAlgorithm/scripts/mps_alisetup.py (hunk 1 above)
Comment (ericvaandering) [inline Alignment/MillePedeAlignmentAlgorithm/scripts/mps_alisetup.py]: No: 

https://docs.python.org/2/library/functools.html#functools.reduce

Comment (ghellwig) [inline Alignment/MillePedeAlignmentAlgorithm/scripts/mps_alisetup.py]: thanks!

**Comment by ghellwig** [comment]
+1

**Comment by dmitrijus** [comment]
+1

**Comment by davidlange6** [comment]
please test

**Pull request merged**